benchmarks/gc.py

```diff
@@ -0,0 +1,39 @@
+from benchmarks.base import BaseRemoteBench
+
+
+class GCBench(BaseRemoteBench):
+    repeat = 1
+    timeout = 12000
+
+    def setup(self):
+        super().setup()
+
+        self.gen("data", "cats_dogs")
+        self.dvc("add", "data", "--quiet")
+        self.dvc("push", "data", "--quiet")
+
+        # remove everything from the local system
+        self.dvc("remove", "data.dvc")
+        self.dvc("gc", "-w")
+
+    def time_gc(self):
+        self.dvc("gc", "-f", "-w", proc=True)
+
+
+class CloudGCBench(BaseRemoteBench):
+    repeat = 1
+    timeout = 12000
+
+    def setup(self):
+        super().setup()
+
+        self.gen("data", "large")
```

Comment:
`gc -c` is terribly slow right now (since there is no concurrency, it would probably take hours for a large dataset like cats_dogs), so I went with a smaller dataset (even this might be much for, but let's see the results before changing it).